com.unity.robotics.urdf-importer/Runtime/Extensions/UrdfSensorExtension.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using UnityEngine;
+using Unity.Robotics.Sensors;
```

Comment:
No, the dependency does not live in the URDF-Importer package and yes it will break any project that uses URDF-Importer and not RoboticsSensors. That is why it is being mentioned a separate dev-sensors branch and is not planned to be merged to dev before RoboticsSensors is released. And even afterwards it has to be enclosed within preprocessor directives for projects that will use URDF-Importer without sensors.